Flat/interface/PandaAnalyzer.h

```diff
@@ -276,6 +299,8 @@ public :
     TF1* puppisd_corrGEN=0;
     TF1* puppisd_corrRECO_cen=0;
     TF1* puppisd_corrRECO_for=0;
+    RoccoR rochesterCorrection;
```

Comment:
Why not make these pointers that are constructed on-demand, e.g. if `Analysis::complicatedLeptons` is set? Would be more uniform with how we do things elsewhere.

Reply:
Agree